core/src/test/resources/util/test-application-multi.yml

```diff
@@ -0,0 +1,29 @@
+spring:
+  profiles:
+    active: dev,qa
+
+---
+
+spring:
+  profiles: dev
```

Comment:
would be good to have a test with multiple, comma separated profiles, too (as I suspect this is currently broken).